tests/keras_tests/layer_tests/test_layers_runner.py

```diff
@@ -285,6 +285,14 @@ def test_averagepooling2d(self):
                             AveragePooling2D(strides=2),
                             AveragePooling2D(strides=(2, 1))]).run_test()
 
+    def test_combinednms(self):
```

Comment:
Same regarding exporting the tests to a separate package

Reply:
A separate package was opened for custom layers. This a native TF layer